docs/model.md

```diff
@@ -1,16 +1,49 @@
 # Model
 
-OTG is an actively developing specification, with contributions coming from real [use cases](/examples/#use-cases). On high level, the model allows to express the following building blocks of a traffic generator configuration:
+## Formal Model
+The formal [model specification](https://github.com/open-traffic-generator/models/blob/master/artifacts/openapi.yaml) can be found on GitHub under [Open Traffic Generator](https://github.com/open-traffic-generator) organization. To explore the model, a viewer friendly [**ReDoc rendering**](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/open-traffic-generator/models/master/artifacts/openapi.yaml) is available as well. The OTG APIs support both REST and gRPC interfaces.
+
+## Building Blocks
+
+OTG is an actively developing specification, with contributions from real [use cases](/examples/#use-cases). The model allows to express the following building blocks of a traffic generator configuration:
+
+* **Test Ports** with Layer 1&2 capabilities, including:
+	- LLDP, LAG, LACP
+* Emulated **OTG Devices** with Layer 2&3 features:
+	- IPv4, IPv6 interfaces
+	- ARP, IPv6 ND
+	- BGP, IS-IS routing protocols
+* **Traffic Flows** 
+    - associated with either Test Ports, or OTG Devices
+	- expressing L2-4 properties like Ethernet, IPv4/IPv6, TCP/UDP
+	- stateless or stateful capabilities for transport protocols
+	- with implementation-specific application payload
+* Run-time **Metrics** and traffic **Capture** capabilities
+
+## Hierarchy
+
+The hierarchy of objects the OTG Model is visualized below.
+<!-- TODO replace with an image from the images subfolder -->
+![OTG Hierarchy](https://raw.githubusercontent.com/open-traffic-generator/models/docs/docs/overview.drawio.svg)
+<p style="text-align: center;"><sub>Fig. 1. Hierarchy of the OTG objects</sub></p>
+
+##Raw Traffic Flows
  
-* *Test Ports* with:
-	- LLDP
-	- LAG
-	- LACP
-* *Traffic Flows* associated with:
-	- Test Ports
-	- Emulated Devices
-* *Emulated Devices* with:
-	- BGP
-	- IS-IS
-
-The specification is available on GitHub [here](https://github.com/open-traffic-generator/models/blob/master/artifacts/openapi.yaml) (viewer friendly version at reDocly [here](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/open-traffic-generator/models/master/artifacts/openapi.yaml)).  Specifications support both REST or gRPC interfaces.
+In the most simple cases, the OTG Model describes **Raw Traffic Flows**: stateless streams of packets to be transmitted from one Test Port, and expected to be received on another Test Port.
```

Comment:
1. "In the most simple case, the OTG Model describes Raw Traffic Flows:..."
2. "...from one Test Port, and typically, though not always, expected to be received on another Test Port" - what about this which indicates that one could do a one-arm test if one wanted.

Reply:
done, with more added to explain two-arm vs one-arm